source/userguide/kb.rst

```diff
@@ -813,3 +813,122 @@ We can perform the same search using the :meth:`query()` API as well.
+  | embedder_keyword      | For leveraging deep semantic embedders and text signals for structured search on keywords or short spans   |
+  |                       | of text. Glove as an additional signal has shown improvements in this use case.                            |
+  +-----------------------+------------------------------------------------------------------------------------------------------------+
+  | embedder_text         | For leveraging deep semantic embedders and text signals for unstructured search on larger paragraphs or    |
+  |                       | passages. BERT as an additional signal has shown improvements in this use case.                            |
+  +-----------------------+------------------------------------------------------------------------------------------------------------+
+
+
+The two included embedder types are `Sentence BERT <https://github.com/UKPLab/sentence-transformers>`_ and `GloVe <https://nlp.stanford.edu/projects/glove/>`_. 
+
+Sentence BERT is a modification of the pretrained BERT network that use siamese and triplet network structures to derive semantically meaningful sentence embeddings that can be compared using cosine-similarity. To use this embedder type, you can specify ``bert`` as the ``embedder_type`` in the ``model_settings``.
+
+Multiple models fine-tuned on BERT / RoBERTa / DistilBERT / ALBERT / XLNet are provided. The available models are specified `here <https://github.com/UKPLab/sentence-transformers#english-pre-trained-models>`_, and the specific trained model can be specified via the ``model_name`` parameter in the ``model_settings`` section of the config. The default is ``bert-base-nli-mean-tokens`` which is a BERT model trained on the `SNLI <https://nlp.stanford.edu/projects/snli/>`_ and `MultiNLI <https://cims.nyu.edu/~sbowman/multinli/>`_ datasets with mean-tokens pooling.
+
+The provided GloVe embeddings are word vectors trained on the `Wikipedia <https://dumps.wikimedia.org/>`_ and `Gigaword 5 <https://catalog.ldc.upenn.edu/LDC2011T07>`_ datasets. To use this embedder type, you can specify ``glove`` as the ``embedder_type`` in the ``model_settings``.
+
+The 50, 100, 200, and 300 dimension word embeddings are provided. The desired dimension can be specified via the ``token_embedding_dimension`` in the ``model_settings`` section of the config. For fields with more than one token, word vector averaging is used. 
+
+Once your config file is specified, you can load the Knowledge Base and the embeddings will automatically be generated as specified in the config and stored in your index as dense vectors. Note that you must pass the application path in order for the config to be processed. This can be done via the CLI command with the ``--app-path`` option or via the ``app_path`` parameter for the ``load_kb`` method.
+
+
+.. code-block:: console
+
+  mindmeld load-kb hr_assistant faq_emb data/hr_faq_data.jsonl --app_path ./
```

Comment:
@arushir Does ``--app_path .`` (i.e. without the `/`) not work?

Reply:
That works as well, can update it